### DISCUSSION TITLE
How to use mentions in Suneditor ? There are no clear sources to implement mentions (@mentions) into Suneditor component

### DISCUSSION THREAD

**Discussion by Redhatios** [JiHong88/SunEditor#1333] [General]
I'm trying to implement @mentions into the Suneditor component. It is essential feature to have @mentions in any of the rich text editors. I saw a custom plugin to implement mentions into sun editor, but I couldn't add into the component.

**Comment by JiHong88** [comment]
It will be added in version 3.0.0.
